### PR TITLE
fix(docker): add @cdt/modele-social to main docker image

### PR DESCRIPTION
**Pull request by lionelB**
add missing code-du-travail-modeles package.json to main  docker image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,13 +27,15 @@ COPY ./packages/sources/package.json /app/packages/sources/package.json
 COPY ./packages/slugify/package.json /app/packages/slugify/package.json
 
 
+
```
Comment (m-maillot) [inline Dockerfile]: It's a lot of empty lines 😄

**Review by m-maillot — APPROVED** [review]
Nice catch!

File: Dockerfile (hunk 1 above)
Comment (lionelB) [inline Dockerfile]:
```suggestion

```

File: Dockerfile [hunk 2]
```diff
@@ -34,6 +34,7 @@ COPY ./packages/slugify/package.json /app/packages/slugify/package.json
 COPY ./packages/react-ui/package.json /app/packages/react-ui/package.json
 COPY ./packages/code-du-travail-api/package.json /app/packages/code-du-travail-api/package.json
 COPY ./packages/code-du-travail-frontend/package.json /app/packages/code-du-travail-frontend/package.json
+COPY ./packages/code-du-travail-modeles/package.json /app/packages/code-du-travail-modeles-social/package.json
```
Comment (lionelB) [inline Dockerfile]:
```suggestion
COPY ./packages/code-du-travail-modeles/package.json /app/packages/code-du-travail-modeles/package.json
```

**Pull request merged**